connectors/cli/connector.py

```diff
@@ -108,17 +101,22 @@ async def __create_search_index(self, index_name, language):
         )
 
     async def __create_connector(
-        self, index_name, service_type, configuration, is_native, language
+        self, index_name, service_type, configuration, is_native, language, from_index
     ):
         try:
             await self.es_client.ensure_exists(
                 indices=[CONCRETE_CONNECTORS_INDEX, CONCRETE_JOBS_INDEX]
             )
             timestamp = iso_utc()
 
+            if not from_index:
+                await self.__create_search_index(index_name, language)
+
+            api_key = await self.__create_api_key(index_name)
```

Comment:
Sorry I only just thought of this -- I don't think API key creation is  necessary for native connectors yet. I'll approve in case this code is urgent to merge because of FF, and creating an API key isn't going to break anything.

Reply:
> Sorry I only just thought of this -- I don't think API key creation is necessary for native connectors yet. I'll approve in case this code is urgent to merge because of FF. Creating an API key for a native connector isn't going to break anything.

Nice. I will address this in a different PR.